synchro/__init__.py

```diff
@@ -298,21 +298,15 @@ def __new__(mcls, name, _bases, attrs):
     return type.__new__(_metaclass, str("dummy"), (), {})
 
 
-class Synchro(with_metaclass(SynchroMeta)):
```

Comment:
Can we also remove with_metaclass now?

Reply:
Done

Reply:
These changes cause this error in the synchro tests:
```
 [2022/02/17 01:40:50.485] Traceback (most recent call last):
 [2022/02/17 01:40:50.485]   File "/opt/python/3.7/lib/python3.7/runpy.py", line 183, in _run_module_as_main
 [2022/02/17 01:40:50.485]     mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
 [2022/02/17 01:40:50.485]   File "/opt/python/3.7/lib/python3.7/runpy.py", line 109, in _get_module_details
 [2022/02/17 01:40:50.485]     __import__(pkg_name)
 [2022/02/17 01:40:50.485]   File "/data/mci/25c39eae21d0be14b9a9ad83b5be38eb/src/synchro/__init__.py", line 291, in <module>
 [2022/02/17 01:40:50.485]     class Synchro(meta=SynchroMeta):
 [2022/02/17 01:40:50.485] TypeError: __init_subclass__() takes no keyword arguments
```

Reply:
Fixed, used the wrong argument name